pkg/controller/builder/builder_component.go

```diff
@@ -80,7 +80,7 @@ func (builder *ComponentBuilder) SetResources(resources corev1.ResourceRequireme
 	return builder
 }
 
-func (builder *ComponentBuilder) DisableExporter(disableExporter *bool) *ComponentBuilder {
+func (builder *ComponentBuilder) SetDisableExporter(disableExporter *bool) *ComponentBuilder {
```

Comment:
for builder pattern the function naming convention should be starts with "WithXXX"